conf/grouping_rules.conf.template

```diff
@@ -14,37 +14,62 @@
 # 
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 
-# This file defines a matching table for context data grouping. The table is composed of several mathcing rules. It
-# works by defining a set of fields within the Orion notification and a regular expression to be applied over the
-# concatenation of those fields. If there is a matching then the destination within the matching rule is used as the
-# name of the permanent storage for the data (it will be a file in the case of HDFS, a MySQL table or a CKAN resource).
+# This file defines a set of grouping rules for context data grouping. It works by defining a set of fields within the
+# Orion notification and a regular expression to be applied over the concatenation of those fields. If there is a
+# matching then:
+# - The destination within the rule is used as the name of the storage element (i.e. this will be a file in
+#   the case of HDFS, part of a MySQL table or a CKAN resource).
+# - The notified fiware-sercicePath is replaced by the configured one.
 # 
 # Append as many matching rules as you need, following this format:
 #
-# integer id|comma-separated fields|regex to be applied to the fields concatenation|destination|dataset
+# {
+#     "grouping_rules": [
+#         {
+#             "id": 1,
+#             "fields": [
+#             ],
+#             "regex": "",
+#             "destination": "",
+#             "fiware_service_path": ""
+#         },
+#         ...
+#     ]
+# }
 #
-# The available "dictionary" of fields is:
-#  - entitydId
-#  - entityType
-#  - servicePath
+# Being:
+# - id: A unique unsigned integer-based identifier. Not really used in the current implementation, but could be useful
+#   in the future.
+# - fields: These are the fields that will be concatenated for regular expression matching. The available dictionary of
+#   fields for concatenation is "entityId", "entityType" and "fiwareService". The order of these fields is important
+#   since the concatenation is made from left to right.
+# - regex: Java-like regular expression to be applied on the concatenated fields.
+# - destination: Name of the HDFS file, MySQL table or CKAN resource where the data will be effectively persisted.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ec3abfa8b762ab7f947c7d74f3fb62dbdaa1722a